src/App/CommentCard/CommentCard.css

```diff
@@ -0,0 +1,43 @@
+
+.CommentCard {
+  /* primary colors */
```

Comment:
🗣️ consider creating a variables.css file and importing it in the root of the application, since both will be in the scope of the DOM the variables will still be accessible from the comment card.

Reply:
Tried implementing this in 042b67f81cb5d0f7b2418e2e375a752ae1808f61